### PR TITLE
MBS-13647: Track event art adds in editor added entities

### PR DESCRIPTION
**Pull request by reosarevok**
### Implement MBS-13647

# Description
This adds event art to all the other added entities stats on the user profile page.

# Testing
Manually, by visiting chaban's profile using a ro db tunnel.

### CODE REVIEW

**Review by mwiencek — APPROVED** [review]
LGTMBDNT

File: lib/MusicBrainz/Server/Data/Editor.pm [hunk 1]
```diff
@@ -709,8 +710,8 @@ sub added_entities_counts {
     return $cached_result if defined $cached_result;
 
     my %result = map { $_ => 0 }
-        qw( artist release area cover_art event instrument label place recording
-        releasegroup series work other );
+        qw( artist release area cover_art event event_art instrument label
+        place recording releasegroup series work other );
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Data/Editor.pm:714]:
```suggestion
        qw( artist release area cover_art event event_art instrument label
            place recording releasegroup series work other );
```

**Comment by jesus2099** [comment]
There is something strange. 

On my profile, I have 0 added event cover arts, of course, as it's new.


Usually, 0 entities rows du not show up at all, but here there is an empty row: https://beta.musicbrainz.org/user/jesus2099

Burt strangely not for this user (with no Series): https://beta.musicbrainz.org/user/emmadilemma

**Comment by reosarevok** [comment]
It only shows temporarily because the stats are cached and it's not 0 for `event_art` in the cached object, it's undefined :) I checked locally without cache and your profile doesn't have that row, so it should go away tomorrow unless you add some event art I think.

**Pull request merged**